holoviews/plotting/bokeh/util.py

```diff
@@ -176,22 +173,40 @@ def refs(json):
     return result
 
 
-def compute_static_patch(document, models, json=None):
+def get_ids(obj):
+    """
+    Returns a list of all ids in the supplied object.
+    Useful for determining if a json representation contains
+    references to other objects.
```

Comment:
Might be worth mentioning why this knowing if the JSON has references is useful information (if it can be summarized in a simple way).